addons/website/static/tests/tours/edit_translated_page.js

```diff
@@ -0,0 +1,21 @@
+odoo.define('website.edit_translated_page', function (require) {
+'use strict';
+
+const tour = require("web_tour.tour");
+
+tour.register('edit_translated_page_redirect', {
+    test: true,
+    url: '/fr_FR',
+},
+[
+    {
+        content: 'click edit master',
+        trigger: 'a[data-action=edit_master]',
+    },
+    {
+        trigger: '#oe_snippets',
```

Comment:
missing `content` property, I even think runbot will crash without it.
it helps greping the line when it crashes